routes/authRouter.js

```diff
@@ -1,10 +1,16 @@
 const router = require('express').Router()
 const sellerController = require('../controllers/seller/index')
-
+const { editStore } = require('../controllers/seller/index')
+const authentication = require('../middlewares/authentication')
 // @route POST api/auth/register
 // @desc Register new user
 // @access Public
 router.post('/register', sellerController.register)
 router.post('/login', sellerController.login)
 
+// @route POST api/auth/edit
+// @desc Edit a seller's store
+// @access Public
+router.put('/stores', authentication, editStore)
```

Comment:
@aapetsi endpoint for editing sellers should not be in authRouter.js

Reply:
Yes I realised. Tolu suggested we have a folder clean up once everything has been merged